components/kubeflow/kfserving/src/kfservingdeployer.py

```diff
@@ -45,54 +45,50 @@ def yamlStr(str):
         return yaml.safe_load(str)
 
 
-def EndpointSpec(framework, storage_uri, service_account):
-    if framework == "tensorflow":
-        return V1alpha2EndpointSpec(
-            predictor=V1alpha2PredictorSpec(
-                tensorflow=V1alpha2TensorflowSpec(storage_uri=storage_uri),
-                service_account_name=service_account,
+def EndpointSpec(framework, storage_uri, service_account, min_replicas, max_replicas):
+
+    endpointSpec = V1alpha2EndpointSpec(
+        predictor=V1alpha2PredictorSpec(
+            service_account_name=service_account,
+            min_replicas=(min_replicas
+                          if min_replicas >= 0
+                          else None
+                         ),
+            max_replicas=(max_replicas
+                          if max_replicas > 0
```

Comment:
We may also want to check max_replicas >= min_replicas.

Reply:
resolved